### ISSUE TITLE
Silent Hill Arcade AMD fix dont seem to work

### ISSUE THREAD

**Issue by orderorder** [teknogods/TeknoParrot#22]
In several levels there are still invisible enemies that take damage. For example, level 3-2-2 after playing for a while. Damage is dealt to the left with no enemies on the screen.

PC:

Ryzen 7 5000
RTX3070m
Windows 10

**Comment by dante3732** [comment]
Yes, the game is only partially fixed, there are still a few invisible enemies left.
